### PR TITLE
Updated all cloud_nodejs git url from feedhenry to feedhenry-templates

### PR DESCRIPTION
**Pull request by luigizuccarelli**
@david-martin please review

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -1061,19 +1061,19 @@
         "name": "Salesforce Connector",
         "type": "other",
         "image": "public/img/cloud_plugins/salesforce.png",
-        "category": "SaaS Connectors",
-        "docs": "https://raw.github.com/feedhenry/fh-connector-salesforce-cloud/master/README.md",
+        "catetesgory": "SaaS Connectors",
```
Comment (david-martin) [inline global.json]: typo?

Comment (luigizuccarelli) [inline global.json]: Yep - created another pr

File: global.json [hunk 2]
```diff
@@ -1061,19 +1061,19 @@
         "name": "Salesforce Connector",
         "type": "other",
         "image": "public/img/cloud_plugins/salesforce.png",
-        "category": "SaaS Connectors",
-        "docs": "https://raw.github.com/feedhenry/fh-connector-salesforce-cloud/master/README.md",
+        "catetegory": "SaaS Connectors",
```
Comment (david-martin) [inline global.json]: @luigizuccarelli different typo :)
`category`

**Comment by jasonmadigan** [comment]
https://github.com/feedhenry/fh-template-apps/pull/122

**Pull request closed without merging**